src/App.css

```diff
@@ -2,3 +2,6 @@
   text-align: center;
 }
 
+.Toastify__toast-container{
+  font-size: 1.5rem;
+}
```

Comment:
Is this BEM naming convention? Can you recheck your syntax.

Reply:
This class is present in the toastify component. Not my custom class.

Reply:
ok